2022-08-07-qubes-os-summit-history.md

```diff
@@ -0,0 +1,97 @@
+
+# How did all of this start?
+
+In May 2019 3mdeb funded Linux training for its employees, having no idea who
+the instructor will be. At some point, it occurred that the mysterious
+instructor will be none other than Marek Marczykowski-Górecki, the same person
+who leads the Qubes OS project. As huge fans of Qubes OS and its approach to
+security, as also fans of Joanna Rutkowska security research in the area of
+firmware and computer architecture, we were delighted to meet him in person in
+our office in Gdańsk, Poland. The time has come when the idea of a short event
+focused on firmware impact on Qubes OS security has been born. We asked Marek
+if he has some spare time to discuss firmware impacts on Qubes OS security and
+he liked the idea. The event has been called humorously a "minisummit". That is
+how the first Qubes OS summit started, one day, in a small conference room,
+with a tiny group of people passionate about firmware and system security,
+nothing outrageous (in the meaning of scale). One may still read what topics
+have been discussed
+[here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/).
+
+# The past of Qubes OS Summits
+
+Everyone enjoyed that one particular day so much, that we decided to repeat the
+event next year. The environment we created together, bending and crossing the
+event horizon between firmware and operating system, was something wonderful,
+both to hear and to speak about. The potential we saw in that kind of event, to
+bring various companies together and collaborate on the project on every
+possible layer, has been pushing us forward to enlarge the community and
+introduce more open-source firmware to the Qubes OS world.
+
+Another May has come, the year 2020, the pandemic dominated the world. But it
+didn't stop us from holding the event again! It even gave us a better
+opportunity to attract and gather more people virtually to attend the event.
+Thanks to our 3mdeb colleagues, who set up the live recording and streaming of
+the whole Qubes OS 2020 Summit, it is possible to see and listen to the talks
+given by Qubes OS and 3mdeb experts on
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIwhPXcuYKtS50CHQOvt_BO)
+over and over again. It was quite a challenge for us because we have organized
+such a virtual event for the first time. Fortunately, everything went smoothly
+and finished with success after 4 sessions 2.5 hours long.
+
+In August 2021 we held yet another Qubes OS Summit together with the Qubes OS
+team, still virtually, unfortunately. However, this year brought more speakers
+from other companies and projects, which we were very happy about. There were 2
+sessions almost 4 hours each and one may watch the talks at
+[Youtube](https://www.youtube.com/playlist?list=PLuISieMwVBpIoLQzpYeZnkupURheXky6r).
+
+# The future of Qubes OSe Summits
+
+Finally, the time has come when we may meet together in person, after two
```

Comment:
two? not three? in person event was in 2019, we saw Marek during FOSDEM 2020 mabe, so at least `over two years`. When I'm counting my personal attendence last time it was Embedded World in February 2020.

Reply:
Yes, three. Fixed.